exampleapp/src/main/res/values/strings.xml

```diff
@@ -3,6 +3,7 @@
 
     <string name="hello_world">Hello world!</string>
     <string name="action_settings">Settings</string>
+    <string name="prefix">ESP8266-</string>
```

Comment:
Seems like this value is specific to your project.  (Or did you intend this as an example of setting a prefix?)